_data/ome2021.json

```diff
@@ -113,7 +113,7 @@
       "alternateName": "Research Data Management",
       "url": "/events/ome-community-meeting-2021/day2/",
       "eventAttendanceMode": "https://schema.org/OnlineEventAttendanceMode",
-      "description": "TBD",
+      "description": "The Research Data Management (RDM) day aims to cover (and perhaps answer) the gamut of pre-publication struggles, from structuring FAIR meta-/data through discovering insights and sharing those with others, all in preparation for day 3 -- public resources.",
```

Comment:
1. gamut ? (still need to check what that means)
2. ``...discovering insights....`` insights are usually "gained" or somesuch ?

Not stuck on the suggestions above, any simplification of the complexity of the sentence would do imho.

Reply:
Pushed